src/app/pages/Designathons/Designathon24/components/Splash/index.jsx

```diff
@@ -0,0 +1,106 @@
+import { memo } from 'react';
+
+const Splash = memo(() => {
+	return (
+		<svg
+			width='3840'
+			height='3840'
+			viewBox='0 0 3840 3840'
+			fill='none'
+			xmlns='http://www.w3.org/2000/svg'
+			xmlnsXlink='http://www.w3.org/1999/xlink'
+		>
+			<g clip-path='url(#clip0_409_136)'>
+				<rect width='3840' height='3840' fill='#000024' />
+				<g filter='url(#filter)'>
+					<path
+						d='M2002.49 1366.5C2369.4 1330.11 2351.34 765.668 2304.45 616.973L2047.91 344.354L-76.1036 -493.379L-1615.7 1084.37C-1647.53 1691.39 -1467.98 3011.3 -495.195 3434.81C720.791 3964.2 -316.92 2499.1 421.501 2276.6C1159.92 2054.1 1059.07 2117.83 1324.11 1968.17C1589.16 1818.52 1635.59 1402.89 2002.49 1366.5Z'
+						fill='#FFCC45'
+					>
+						<animateTransform
+							attributeName='transform'
+							attributeType='XML'
+							type='translate'
+							values='0 0; 50 50; 0 0'
+							dur='20s'
+							repeatCount='indefinite'
+						/>
+					</path>
+				</g>
+				<g filter='url(#filter1_f_409_136)'>
+					<circle cx='1750.9' cy='248.5' r='1039.5' fill='#011EFB' />
+				</g>
+				<g style={{ mixBlendMode: 'overlay' }} filter='url(#filter)'>
+					<path
+						d='M1282.36 1562.48C1845.68 1629.13 1538.71 917.777 2472.06 1289.48C2526.66 1334.5 2625.63 1486.42 2584.69 1733.97C2533.51 2043.42 2080.22 2220.24 958.633 2520.57C61.3642 2760.83 -252.503 1788.22 -297.278 1271.89C-297.46 1118.98 -215.096 804.686 115.812 770.816C529.448 728.477 719.04 1495.83 1282.36 1562.48Z'
+						fill='#089AFF'
+					>
+						<animate
+							attributeName='fill'
+							values='#089AFF;#088Add;#089AFF'
```

Comment:
can we actually remove all the contents of this splash and replace it with a bland div of any accent color that elise has chosen? this filter is actually extremely unperformant on firefox and since we're changing this anyways, no need to bring this over